src/python/pants/option/options_integration_test.py

```diff
@@ -1,315 +1,96 @@
-                "options",
-                "--skip-inherited",
-                "--name=colors",
-            ]
+    )
+    with setup_tmpdir({"plugins/mock_options/register.py": plugin}) as tmpdir:
+        config = {
+            "GLOBAL": {
+                "pythonpath": [f"%(buildroot)s/{tmpdir}/plugins"],
+                "backend_packages": ["mock_options"],
+            },
+            "mock-options": {"deprecated": "foo"},
+        }
+        result = run_pants(["help"], config=config)
+        result.assert_success()
+        assert (
+            "DEPRECATED: option 'deprecated' in scope 'mock-options' will be removed in version "
+            "999.99.9.dev0."
+        ) in result.stderr
+
+        # Now use `ignore_pants_warnings`.
+        config["GLOBAL"]["ignore_pants_warnings"] = ["DEPRECATED: option 'deprecated'"]  # type: ignore[index]
+        ignore_result = run_pants(["help"], config=config)
+        ignore_result.assert_success()
+        assert "DEPRECATED: option 'deprecated'" not in ignore_result.stderr
+
+
+def test_pants_symlink_workdirs() -> None:
+    with temporary_dir() as tmp_dir:
+        symlink_workdir = f"{tmp_dir}/.pants.d"
+        physical_workdir_base = f"{tmp_dir}/workdirs"
+        physical_workdir = f"{physical_workdir_base}/{safe_filename_from_path(symlink_workdir)}"
+
+        pants_run = run_pants_with_workdir(
+            [f"--pants-physical-workdir-base={physical_workdir_base}", "help"],
+            workdir=symlink_workdir,
         )
         pants_run.assert_success()
-        unstripped_lines = (s.split("(", 1)[0] for s in pants_run.stdout.split("\n") if "(" in s)
-        lines = [s.strip() for s in unstripped_lines]
-        # This should be included because it has no super-scopes.
-        self.assertIn("colors = False", lines)
-        # These should be included because they differ from the super-scope value.
-        self.assertIn("setup-py.colors = True", lines)
-        # These should be omitted because they have the same value as their super-scope.
-        self.assertNotIn("pytest.colors = False", lines)
-
-    def test_pants_ignore_option(self) -> None:
-        with temporary_dir(root_dir=os.path.abspath(".")) as tempdir:
-            config_path = os.path.relpath(os.path.join(tempdir, "config.toml"))
-            with open(config_path, "w+") as f:
-                f.write(
-                    dedent(
-                        """
-                        [GLOBAL]
-                        pants_ignore.add = ['some/random/dir']
-                        """
-                    )
-                )
-            pants_run = self.run_pants(
-                [f"--pants-config-files={config_path}", "--no-colors", "options"]
-            )
-            pants_run.assert_success()
-            self.assertIn(
-                f"pants_ignore = ['.*/', '/dist/', 'some/random/dir'] (from CONFIG in {config_path})",
-                pants_run.stdout,
-            )
```

Comment:
Idk why this was a dedicated test. Afaict, we're just testing that adding to a list option works..